guesser.rb

```diff
@@ -0,0 +1,62 @@
+#global vars
+is_game_active = true
+is_new_game = true
+play_again = nil
+lower_bound = 1
+upper_bound = 100
+
+while is_game_active
+
+  if is_new_game
+    puts "Enter a number between 1 and 100"
+    random_num = gets.chomp
```

Comment:
❕ This is actually not required for this program. The user need only _think_ of a number between 1-100, but should not input it.

Remove this logic so you can skip the additional loops and conditionals.